paper/paper.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Picasso: Performance Portable Particle-in-Cell'
+tags:
+  - C++
+  - Kokkos
+  - Cabana
+  - particle-in-cell
+  - material point method
+authors:
+  - name: Stuart Slattery^[corresponding author]
+    orcid: 0000-0003-0103-888X
+    affiliation: 1
+  - name: Samuel Temple Reeve
+    orcid: 0000-0002-4250-9476
+    affiliation: 1
+  - name: Austin Isner
+    affiliation: 1
+  - name: Kwitae Chong
+    affiliation: 1
+  - name: Lance Bullerwell
+    affiliation: 1
+affiliations:
+  - name: Oak Ridge National Laboratory, Oak Ridge, TN, USA
+    index: 1
+date: 12 April 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+Particle-in-cell (PIC) methods are used is disparate fields, from plasma
+physics to solid mechanics and computer graphics. 
+`Picasso` is a performance portable library for PIC simulations, developed
+throughout the Exascale Computing Project (ECP) [@ecp:2020]. `Picasso` builds
+on the `Cabana` library [@cabana:2022] for particle and structured grid
+methods, which in turn extends the `Kokkos` library for on-node parallelism
+across hardware architectures [@kokkos:2022] and `MPI` for scalable multi-node
+communication. `Picasso` provides interpolation schemes for particle-to-grid
+and grid-to-particle updates, embedded free surface tracking, and robust
+linear algebra support for particle and grid operations.
+
+# Statement of need
+
+
+# Library capabilities
+
+## Interpolation
+
+
+## Free surfaces
+
+
+## Examples and performance testing
+
```

Comment:
@kwitaechong is writing the example on the dam break problem and merging to repo. I will help him writeup when complete